docs/debugging.md

```diff
@@ -0,0 +1,192 @@
+<<<<0: 200 OK
+  { "version": { "major": 13, "minor": 0, "additional_info": "release" },
+    "network_version":
+      { "chain_name": "TEZOS_JAKARTANET_2022-04-27T15:00:00Z",
+        "distributed_db_version": 2, "p2p_version": 1 },
+    "commit_info":
+      { "commit_hash": "cb9f439e58c761e76ade589d1cdbd2abb737dc68",
+        "commit_date": "2022-05-05 11:55:24 +0200" } }
+>>>>1: http://localhost:8732/chains/main/blocks/head/protocols
+<<<<1: 200 OK
+  { "protocol": "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY",
+    "next_protocol": "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY" }
+>>>>2: http://localhost:6732/authorized_keys
+<<<<2: 200 OK
+  {}
+>>>>3: http://localhost:6732/keys/tz1g7VaasrnCUmSmCKUcP8o61vMPscVkoxYL
+  "05deadbeef"
+<<<<3: 200 OK
+  { "signature":
+      "edsigu5xkT8xcTEECLri3WZG2QuFLgH47NgFaRKQpBnaUwPw9DfhKHSNTAcvLxqVqWc4i7SEiUciHcEUzmeKYQDQioKcBzayUaY" }
+Signature: edsigu5xkT8xcTEECLri3WZG2QuFLgH47NgFaRKQpBnaUwPw9DfhKHSNTAcvLxqVqWc4i7SEiUciHcEUzmeKYQDQioKcBzayUaY
+$
+```
+
+## Octez tezos-signer
+
+The Octez tezos-signer is the signer of reference for Tezos. Signatory and tezos-signer implement the same core signing capabilities. Running `tezos-signer` for comparative purposes can be helpful when testing or developing signatory. 
+
+## Getting tezos-signer
+
+Download the latest `tezos-signer` binary from the _Releases_ page at https://github.com/serokell/tezos-packaging
+You will also want the `tezos-client` binary for testing & verification purposes.
+
+## Setup tezos-signer with a local secret
+
+To generate a new secret using `tezos-signer` run the command:
+
+```
+tezos-signer gen keys octez_signer_key
+```
+
+This will create (or add to) the configuration files in `~/.tezos-signer`
+
+:::tip
+You can use many of the familiar `tezos-client` sub-commands with the `tezos-signer` such as;
+
+`tezos-signer list known addresses` and `tezos-signer show address octez_signer_key`
+::: 
+
+Start the `tezos-client` in `http` listener mode.
+
+```
+tezos-signer launch http signer
+```
+
+With this secret in place, you can now add this new remote signer to your `tezos-client` configuration by running the command. (Don't forget to replace PKH with the public key hash of your newly generated key in the previous steps)
+
+```
+tezos-client import secret key octez_signer_remote_key http://localhost:6732/${PKH}
+```
+
+You can now test that this key works for signing purposes by using the `tezos-client sign` command. This command will send the value `0x00` from `tezos-client` to the remote signer. Adding the `--log-requests` flag is helpful for debugging.
+
+```
+tezos-client sign bytes 0x00 for octez_signer_local_key
+```
+
+## Setup tezos-signer with a ledger device
+
+Ledger devices have two applications for Tezos, an interactive "Tezos Wallet" app, and a "Tezo Baking" app which is designed for unattended use when operating a Tezos baker on the Tezos network.
+
+
+### Tezos Baking App
```

Comment:
The extra # to make it a smaller heading is messing with the indentation in the sidebar. I don't know what is causing it to happen but we may want to rethink having this heading

<img width="176" alt="image" src="https://user-images.githubusercontent.com/29528321/175126538-0604c39b-8dc9-4f3c-8ea0-5c6ba2c9a039.png">

Reply:
ok